pages/hydra-agent-redis-graph.md

```diff
@@ -4,6 +4,235 @@ title: Redis as a graph database for Hydra Agent | Hydraecosystem.org
 permalink: /hydra-agent-redis-graph
 ---
 
+# Introduction:
+
+The hydra-python-agent is a smart Hydra client implemented in Python which works with[ Hydrus](https://github.com/HTTP-APIs/hydrus). Smart clients are generic automated clients that establish resilient connected data networks leveraging knowledge graphs.
```

Comment:
The agent works with all hydra based servers not just hydrus

Reply:
Right! I was so used to using Hydrus that it came out naturally. I have fixed it now.

Reply:
@xadahiya ^^